Agsv2 Port

dianaw353 opened the pull request:
This is port for agsv2 and it will be a work in progress for a while

**AGSV2 Bar**
- [ ] Taskbar
  - [x] App Launcher - end of year
    - [x] Title
    - [x] configurable Icon
  - [x] Workspaces
  - [x] App Title
  - [ ] Time/Date - end of year
    - [ ] Time
    - [ ] Notifications
      - [ ] Clear button
    - [ ] Calender
  - [ ] Music
    - [x] Music title
    - [ ] Hides title after x time or when clicked
  - [ ] Quick settings - end of year
    - [ ] Wifi
    - [ ] Sound
    - [ ] Bluetooth
    - [ ] Brighness
    - [ ] Power settings
    - [ ] dark/light mode - can be finished next  year...
    - [ ] mic
    - [ ] do not distrb mode
  - [ ] Battery
    - [x] Icon and precentage
    - [ ] Notification Warnings
  - [ ] Hyprpicker - end of year
  - [x] Power Menu
    - [x] Shutdown 
    - [x] Reboot
    - [x] Sleep
    - [x] Logout
    - [x] Lock 
  - [ ] OSD - hopefully by end of year
    - [ ] Sound
    - [ ] Brightness
    - [ ] Touchpad
    - [ ] Mic

**AGSV2 features**
 - [ ] Notification when x package is not installed when clicked eg. hyprpicker
 - [ ] Sceenshots
   - [ ] full screen
   - [ ] partial screen
   - [ ] video
   - [ ] #44 
 - [ ] Themes - light/dark to start out with
 - [ ] keybinds cheatsheet
   - [ ] Reads the active file location for keybinds and reads the content for keybinds and organize them into a nice fashion
 - [ ] Greetd login screen


  **Dotfiles**
  - [ ] Download https://github.com/dianaw353/wallpapers - end of year
  - [ ] Spotify with spiceify mods and theme - end of year
  - [ ] Basic flathub support - end of year
  - [ ] Migrate to only use agsv2 - everything else needs to be finished before this

dianaw353 committed:
agsv2 init

dianaw353 committed:
styling refactor

dianaw353 committed:
style cleanup

dianaw353 committed:
minor styling improvements

dianaw353 committed:
remove slider on bar

dianaw353 committed:
better widget orginization

dianaw353 committed:
add notification widget

dianaw353 committed:
scss restructure

dianaw353 committed:
deleted duplicated files & add applauncher

dianaw353 committed:
add applauncher to bar

dianaw353 committed:
cleanup & minor improvements

dianaw353 committed:
media tweaks & border width increase

dianaw353 committed:
remove .Cover css

dianaw353 committed:
use varables for most things

dianaw353 committed:
merge all variables into dark_variables

dianaw353 committed:
power menu added

needs themeing improvements but basics are done :3

dianaw353 committed:
remove unneeded file

dianaw353 committed:
remove comments

dianaw353 committed:
sytle changes

dianaw353 committed:
enable notifications to disapper after x time

kurtbahartr committed:
Set AGS version to both

kurtbahartr committed:
Update the warning in README

We're at dev branch.

kurtbahartr committed:
Revamp the designs for power and apps menus in AGSv2

kurtbahartr committed:
Make AGSv2 power menu dismissable by clicking outside & proper ident

kurtbahartr committed:
Merge branch 'main' of https://github.com/dianaw353/dotfiles into dev

kurtbahartr committed:
Also possess "both" AGS version config for v2-by-default

kurtbahartr committed:
Make it clear that we're on a development branch, not a regular ver

kurtbahartr committed:
Merge branch 'main' of https://github.com/dianaw353/dotfiles into dev

kurtbahartr committed:
Add adw-gtk-theme and an option to switch between GTK3 and GTK4 Adw

kurtbahartr committed:
Move gnome-themes-extra to pre-tasks

kurtbahartr committed:
Also install Eye of GNOME

kurtbahartr committed:
Fix icon substitude for OpenJDK 23

kurtbahartr committed:
Make favorites in AGSv1 configurable

kurtbahartr committed:
Make keybound apps configurable

kurtbahartr committed:
Use an AUR helper instead of Pacman for kernel headers package

kurtbahartr committed:
Add optional USE_MPERFORMANCE flag

kurtbahartr committed:
Install nvidia-dkms instead of nvidia

kurtbahartr committed:
Add an option for default JDK version

dianaw353 commented on group_vars/all.yml:
make sure u can have multiple versions listed here

fkjbvfdvbnfdkjvbjfhd replied:
If you mean for AGS version, sure.

If you mean for the default Java version, `archlinux-java set` accepts only one Java version as the default. That doesn't mean only the default will be installed tho, it's still going to install all four versions.

kurtbahartr committed:
`system.ags.version` is now `system.ags.versions` as a list

kurtbahartr committed:
Tweak the Apps button to include Arch logo

The design changes I brought along also fixes the overall margins
that were too tight before.

kurtbahartr committed:
Fix the power menu not dismissing when clicked certain areas outside

kurtbahartr committed:
Fix artist name showing the previous artist when seeking media

kurtbahartr committed:
Make notification dismiss button rounded too

kurtbahartr committed:
Make action buttons in the power menu rounded

kurtbahartr committed:
Fix the warnings for division in PowerMenu style

kurtbahartr committed:
Fix the weird enormous padding between texts

dianaw353 committed:
remove developer drivers for amd

these drivers are not needed for daily tasks
